src/pages/Opportunities.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from "react";
+import { ChevronLeftIcon, ChevronRightIcon } from "@heroicons/react/20/solid";
+
+function Opportunities() {
+  const [activeCategories, setActiveCategories] = useState([]);
+
+  const buttons = [
+    { text: "All Issues" },
+    { text: "Category 1" },
+    { text: "Category 2" },
+    { text: "Category 3" },
+  ];
+
+  const handleButtonClick = (buttonText) => {
+    if (buttonText === "All Issues") {
+      setActiveCategories([]);
+    } else {
+      setActiveCategories((prevCategories) =>
+        prevCategories.includes(buttonText)
+          ? prevCategories.filter((category) => category !== buttonText)
+          : [...prevCategories, buttonText]
+      );
+    }
+  };
+
+  const opportunitiesList = [
+    {
+      id: 1,
+      title: "2024 Fun Sports Equipment Drive 4 Homeless",
+      image: "url",
+      date: "Sep 16",
+      location: "Los Angeles",
+      description:
+        "Repudiandae sint consequuntur vel. Amet ut nobis explicabo numquam expedita quia omnis voluptatem. Minus quidem ipsam quia iusto.",
+    },
+    {
+      id: 2,
+      title: "2024 Fun Sports Equipment Drive 4 Homeless",
+      image: "url",
+      date: "Sep 16",
+      location: "Los Angeles",
+      description:
+        "Repudiandae sint consequuntur vel. Amet ut nobis explicabo numquam expedita quia omnis voluptatem. Minus quidem ipsam quia iusto.",
+    },
+    {
+      id: 3,
+      title: "2024 Fun Sports Equipment Drive 4 Homeless",
+      image: "url",
+      date: "Sep 16",
+      location: "Los Angeles",
+      description:
+        "Repudiandae sint consequuntur vel. Amet ut nobis explicabo numquam expedita quia omnis voluptatem. Minus quidem ipsam quia iusto.",
+    },
+    {
+      id: 4,
+      title: "2024 Fun Sports Equipment Drive 4 Homeless",
+      image: "url",
+      date: "Sep 16",
+      location: "Los Angeles",
+      description:
+        "Repudiandae sint consequuntur vel. Amet ut nobis explicabo numquam expedita quia omnis voluptatem. Minus quidem ipsam quia iusto.",
+    },
+  ];
+
+  return (
+    <>
+      <section className="bg-slate-200 text-center py-10">
+        <div className="container items-center mx-auto max-w-5xl px-4 py-4">
+          <input
```

Comment:
Add comment that search isn't functional yet, and create Jira ticket to track for future work

Reply:
@justgogame